Server/Controllers/Generation/GenerationAudio.js

```diff
@@ -34,4 +46,111 @@ exports.generationAudio = async function (req, res) {
     const temp = res.status(returnCode).send(returnMessage);
     await axios.post(urlSetStatus, { statusType: returnStatus, stepType: 'AudioGeneration' });
     return (temp);
+}
+
+const getEnd = (audio) => audio.timeStamp + audio.duration;
+
+async function getfiles(projectId, audioInfo) {
+    var updatedAudioInfo = []
+    try {
+        for (let audio of audioInfo) {
+            if (audio.id == undefined || audio.timeStamp == undefined || audio.duration == undefined)
+                throw("invalid object");
```

Comment:
```suggestion
                throw(new Error("Invalid object"));
```
Plutôt un bail comme ça, a voir